extensions.json

```
@@ -1,5 +1,10 @@
 {
   "extensions": [
+    {
+      "id": "0x9ef.vscode-vlang",
+      "repository": "https://github.com/vlang/vscode-vlang",
```

Oh nice, the extension actually provides a `.vsix` release package! In that case I'd prefer to upload that directly, as re-packaging ourselves carries the risk of doing it the wrong way and accidentally uploading a dysfunctional extension.

```suggestion
      "download": "https://github.com/vlang/vscode-vlang/releases/download/0.1.12/vscode-vlang-0.1.2.vsix",
```